packages/ui-text-input/src/TextInput/__tests__/TextInput.test.tsx

```diff
@@ -180,65 +180,6 @@ describe('<TextInput/>', async () => {
     expect(ref?.value).to.equal('bar')
   })
 
```

Comment:
These tests were deleted in the new PR too, in the new version we dont read messages by the SR